tools/ci_build/exclude_unused_ops.py

```diff
@@ -297,23 +301,62 @@ def get_provider_path(ort_root=None, use_cuda=False):
     return provider_paths  # end of get_provider_paths
 
 
+def create_config_file_with_required_ops(required_operators, model_path, config_path, output_file):
+
+    directory, filename = os.path.split(output_file)
+    if not filename:
+        log.error("Invalid path to write final config to. {}".format(output_file))
+
```

Comment:
Do you want to exit after this, or wait until the  open(output_file, 'w') fails?

Reply:
Yep. Good catch.